src/F2_autoreport.py

```diff
@@ -115,11 +115,15 @@ async def _print_pdf_from_chrome(path_pdf_report):
 
 
 def print_pdf(app=None, path_pdf_report=os.path.join(OUTPUT_FOLDER, "out.pdf")):
-    """Run the dash app in a thread an print a pdf before exiting
+    r"""Runs the dash app in a thread and print a pdf before exiting
 
     Parameters
     ----------
     app: handle to a dash app
+        Default: None
```

Comment:
```suggestion
    app: handle to a dash app
        Default: None
```
Is there an official type for the app? "dash app handle"? The definition does not explain what it does

Reply:
the app is a instance of the Dash class of the dash library. I will rewrite it as such :)